mne/utils/docs.py

```diff
@@ -728,6 +728,49 @@
     If True, use a linear transparency between fmin and fmid.
     None will choose automatically based on colormap type.
 """
+
+# STC label time course
+docdict['eltc_labels'] = """
+labels : Label | BiHemiLabel | list of Label or BiHemiLabel
+    The labels for which to extract the time course.
+"""
+docdict['eltc_src'] = """
+src : list
+    Source spaces for left and right hemisphere.
+"""
+docdict['eltc_mode'] = """
+mode : str
+    Extraction mode, see Notes.
+"""
+docdict['eltc_allow_empty'] = """
+allow_empty : bool
+    Instead of emitting an error, return all-zero time courses for labels
+    that do not have any vertices in the source estimate. Default is ``False``.
+"""
+docdict['eltc_mode_notes'] = """
+Valid values for ``mode`` are:
+
+``'max'``
```

Comment:
I usually find that:
```
- X
     desc
- Y
    desc
```
looks better when rendered (otherwise they look like headings rather than a list of options), but I'm fine with this. Feel free to merge if you're happy @drammock

Reply:
addressed by 4e3d1e2